starter-code/public/css/gallery.css

```diff
@@ -0,0 +1,33 @@
+
```

Comment:
You could have put everything into one CSS file and gave class names when needed